From f44b3798c73a32c479c2cd95f4205659c5704549 Mon Sep 17 00:00:00 2001
From: Laurent Pinchart <laurent.pinchart@ideasonboard.com>
Date: Sat, 14 Jan 2023 23:38:53 +0200
Subject: [PATCH 1/9] media: v4l2-subdev: Enable the streams API

The ISI driver depends on the stream API. Enable it manually, until it
stabilizes upstream and gets enabled by default.

Upstream-Status: Inappropriate
Signed-off-by: Laurent Pinchart <laurent.pinchart@ideasonboard.com>
---
 drivers/media/v4l2-core/v4l2-subdev.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/v4l2-core/v4l2-subdev.c b/drivers/media/v4l2-core/v4l2-subdev.c
index 3a4ba08810d24..f535929e7c24d 100644
--- a/drivers/media/v4l2-core/v4l2-subdev.c
+++ b/drivers/media/v4l2-core/v4l2-subdev.c
@@ -32,7 +32,7 @@
  * 'v4l2_subdev_enable_streams_api' to 1 below.
  */
 
-static bool v4l2_subdev_enable_streams_api;
+static bool v4l2_subdev_enable_streams_api = true;
 #endif
 
 /*
-- 
2.47.2

